extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -43,7 +43,8 @@ class AmpAccordion extends AMP.BaseElement {
       log.warn(TAG, `Experiment ${EXPERIMENT} disabled`);
       return;
     }
-    this.sections_.forEach(section => {
+    this.element.setAttribute('role', 'tablist');
```

Comment:
Please ensure that you also add `aria-expanded` to the `tabpannel` (along with the existed `expanded`) as described here: https://www.w3.org/TR/wai-aria/roles#tablist